base_edifact/models/edifact.py

```diff
@@ -242,3 +242,47 @@ def map2odoo_description(self, seg):
             description = seg[2][3]
             return description
         return None
+
+    def _replace_empty(self, value):
+        if value is False:
+            return ""
+        return str(value)
+
+    @api.model
+    def create_segment(self, *elements):
+        cleaned_elements = []
+        for element in elements:
+            if isinstance(element, list):
+                cleaned_elements.append(
+                    [self._replace_empty(value) for value in element]
+                )
+            else:
+                cleaned_elements.append(self._replace_empty(element))
+        return Segment(*cleaned_elements)
+
+    def create_interchange(self, sender, recipient, control_ref, syntax_identifier):
```

Comment:
The docstring should state what the method does

https://peps.python.org/pep-0257/#multi-line-docstrings

Reply:
Thank you @simahawk for your review. I updated it.